packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -203,6 +203,20 @@ contract OPStackManager is ISemver, Initializable {
         bytes32 salt = bytes32(_input.l2ChainId);
         DeployOutput memory output;
 
+        // -------- TODO: Placeholders --------
+        // For contracts we don't yet deploy, we set the outputs to  dummy proxies so they have code to pass assertions.
+        // We do these first, that way the disputeGameFactoryProxy is set when passed to the SystemConfig input.
+        output.disputeGameFactoryProxy = DisputeGameFactory(deployProxy(l2ChainId, output.opChainProxyAdmin, "1"));
```

Comment:
Why is the `_contractName` parameter not the name of the contract? Instead it's 1,2,3,4 etc.

Reply:
Once we actually implement FPs for real (@maurelian will start working on this), then we should change that to use the actual contract name. This is just placeholder code for now (note that this code was not changed in this PR, just moved)